src/test/javascript/spec/component/overview/course-discussion/course-discussion.component.spec.ts

```diff
@@ -401,27 +414,6 @@ describe('CourseDiscussionComponent', () => {
         expect(metisServiceGetFilteredPostsSpy).toHaveBeenCalledTimes(3);
     }
 
-    describe('sorting of posts', () => {
```

Comment:
I don't see a replacement for this test 🤔  Is there one?

Reply:
This test tests functionality that wasn't needed anymore, so I deleted the corresponding function. However, I will add it again, just in case somebody needs it in the future.